src/student/templates/student/students_list_two.html

```diff
@@ -23,7 +23,8 @@
     </div>
     </form>
 
-    {{ students_list|safe }}
+    {% include 'student/includes/students_table.html' %}
+    {% include 'student/includes/pagination.html' %}
```

Comment:
Pagination could be made a part of table template. This simplifies uniforming interface. 
For example, now group form doesn't have pagination for its related students. If pagination were made a part of student table, students in groups would have displayed with pagination.

Reply:
Fixed.